packages/ts-scripts/src/cdk/components/rds-database.ts

```diff
@@ -29,6 +30,13 @@ export const createDatabase = (
 
   db.connections.allowFromAnyIpv4(ec2.Port.allTcp())
 
+  if (bastion) {
+    const bastionInstance = new ec2.BastionHostLinux(stack, 'bastion', {
+      vpc,
+      instanceType: ec2.InstanceType.of(ec2.InstanceClass.T2, ec2.InstanceSize.MICRO),
```

Comment:
Please can you use t4g with a gp3 ebs volume (assuming you can configure the latter). As per https://reapit.atlassian.net/browse/RPLT-449